tests/x11regressions/firefox/firefox_appearance.pm

```diff
@@ -30,16 +30,12 @@ sub run {
     sleep 1;
     type_string "addons.mozilla.org/en-US/firefox/addon/opensuse\n";
     assert_screen('firefox-appearance-mozilla_addons', 90);
-    send_key "alt-f10";
     wait_still_screen 3;
```

Comment:
that should not be necessary anymore if the next step is an "assert_and_click", right?